configs/katello/4.0.yaml

```diff
@@ -0,0 +1,100 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.0.0:
+    :redmine_version_id: 1177
+:prior_releases:
+  :3.18.0:
+    :redmine_version_id: 1272
+  :3.18.1:
+    :redmine_version_id: 1338
+  :3.18.2:
+    :redmine_version_id: 1353
+:code_name:
+:mash_scripts:
+  - katello-4.0-rhel7
+  - katello-4.0-thirdparty-candlepin-rhel7
+  - katello-4.0-el8
+  - katello-candlepin-4.0-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.0
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.4
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.24.2
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.4
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.7.3
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.3
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
```

Comment:
@jjeffers this will need updates, but I wanted to get the basic config layed out so I can branch koji

Reply:
@evgeni ok - I'll need to see what the versions of hammer, et al, are to be. Ty!